nomad/acl_test.go

```diff
@@ -686,120 +686,3 @@ func TestResolveSecretToken(t *testing.T) {
 		})
 	}
 }
-
-func TestResolveClaims(t *testing.T) {
```

Comment:
Note to reviewers: it looks like I'm throwing out a big bunch of tests here, but this was already moved to `nomad/auth/auth_test.go` in the 1st patch in this series, and I should have removed it then. :grinning: